hawkular-metrics/Dockerfile

```diff
@@ -36,6 +36,19 @@ ENV HAWKULAR_METRICS_ENDPOINT_PORT="8080" \
 # The http and https ports
 EXPOSE 8080 8444
 
+# Install Hawkular-Metrics Python client, only available on EPEL
```

Comment:
On this Dockerfile, I mostly just re-arranged things so that "rare" changes are on top, and frequent changes are at the bottom.